packages/gatsby/src/utils/parcel/__tests__/fixtures/error-in-code-ts/gatsby-node.ts

```diff
@@ -0,0 +1,47 @@
+import { GatsbyNode } from "gatsby"
+import { working } from "../utils/say-what-ts"
+import { createPages } from "../utils/create-pages-ts"
+
+this is wrong syntax that should't compile
```

Comment:
Should this line be a comment?

Reply:
Oh or is this there to fail the compilation 👍

Reply:
Yeah, that's test fixture to test behavior when there is error in source file that should result in compilation problem and test itself is here https://github.com/gatsbyjs/gatsby/blob/31e704918a82f3d79bc82c91bfa1e3cb248d3020/packages/gatsby/src/utils/parcel/__tests__/compile-gatsby-files.ts#L180-L210

I did break this behavior when iterating on this code and noticed we didn't have tests for this case so I added it as part of this PR

https://github.com/gatsbyjs/gatsby/pull/38773#discussion_r1433729682 this comment highlight opportunity to improve error message possibly, but this was just out of scope of the fix